packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -566,6 +567,53 @@ export class ConversationService extends TypedEventEmitter<Events> {
     }
   };
 
+  /**
+   * Will try to register mls group by sending an empty commit to establish it.
+   * After group was successfully established, it will try to add other users to the group.
+   *
+   * @param groupId - id of the MLS group
+   * @param conversationId - id of the conversation
+   * @param selfUserId - id of the self user
+   * @param qualifiedUsers - list of qualified users to add to the group (should not include the self user)
+   */
+  public async tryEstablishingMLSGroupWithUsers({
```

Comment:
```suggestion
  public async tryEstablishingMLSGroup({
```

Let's shorten the naming a little. It's explicit that you can add users since it's in the param payload